python/cuml/cuml/ensemble/CMakeLists.txt

```diff
@@ -14,8 +14,6 @@
 
 set(cython_sources "")
 add_module_gpu_default("randomforest_common.pyx" ${randomforestclassifier_algo} ${randomforestregressor_algo} ${ensemble_algo})
-add_module_gpu_default("randomforestclassifier.pyx" ${randomforestclassifier_algo} ${ensemble_algo})
-add_module_gpu_default("randomforestregressor.pyx" ${randomforestregressor_algo} ${ensemble_algo})
```

Comment:
With the new `cuml.ensemble` structure, only the common base class needs to be in cython, the rest is pure python.